### ISSUE TITLE
delete graph data value and update min or max

### ISSUE THREAD

**Issue by bdwilton** [domoticz/domoticz#1691]
v3.0872

It would be nice if after deleting a data point on a graph that the statistics could get updated to remove in my case the low bogus value ?

**Comment by gizmocuz** [comment]
First, why is this low bogus value being added... if you are testing it, then just recreate the device
What Sensor type/screenshot?

**Comment by bdwilton** [comment]
happens on a dht22 sensor I have connected to rPi unit but are querying data through a script, I also get it from time to time from a bme280 running on a ESP12-f (EspEasy firmware). Basically just a bogus value of 0 degrees on the ESP and a similar rubbish value on the dht22.

It could get fixed in the respective TX code but I came across the delete feature (shift delete) on a graph and thought it was worth mentioning that a delete of a low value it would be nice if it also fixed the statistics, if the low or high of the statistics is matched.

Possibly not correct as the stats are for the month and year so could be time consuming.  Could be a seperate menu option to check statistics or tick box I guess.

I should spend some time getting familiar with the code.

**Comment by gizmocuz** [comment]
You are using a script to poll and publish this to domoticz ?
Did you try with the internal build-in hardware support for them (i2c) ? i think in those classes there are special checks for these values

**Comment by bdwilton** [comment]
Yes, in this case its a bash script calling python.  I didn't see at the time a direct option for i2c to DHT22 with domoticz, I did have a BME280 running that way but that unit is going outside as mentioned with the ESP12-f unit.  I think one of the issues with corruption in this case is electrical interferance with another sensor on the same Cat5, better since I GND all pairs though.  I'll see if i2c is an option.  

I have a second rPi unit running in the garage with remote GPIO enabled with a second dht22 unit.  This is probably a second topic but with raspian remote gpio enabled there is a pigpio library daemon that is enabled allowing GPIO from the master rPi unit.  This is a C Library but has python units, I have tried the DHT22.py unit they have and it seems to work correctly.  Just wondering if this library has a Domoticz interface / intergration.

Console test as mentioned works fine, I'd just need to add the publish to domoticz.

Reference for library is http://abyz.co.uk/rpi/pigpio/
